public/app/features/admin/UserOrgs.tsx

```diff
@@ -155,8 +161,9 @@ class UnThemedOrgRow extends PureComponent<OrgRowProps> {
     }
   }
 
-  onOrgRemove = () => {
-    const { org } = this.props;
+  onOrgRemove = async () => {
+    const { org, user } = this.props;
+    user && (await updateUserRoles([], user.id, org.orgId));
```

Comment:
what happens if this first request fails?

wondering if it is better to implement to remove the permissions within the same api call to avoid bad states, where one of these api calls fails.
@briangann wdyt?

@kalleep maybe you have something better in mind 🤔

Reply:
general error handling needs work, previously it would leave roles behind for deleted accounts too, and when you went to add them back in, the old roles appeared "out of nowhere"